smriprep/workflows/surfaces.py

```diff
@@ -531,9 +529,21 @@ def init_gifti_surface_wf(*, name="gifti_surface_wf"):
         run_without_submitting=True,
     )
     fs2gii = pe.MapNode(
-        fs.MRIsConvert(out_datatype="gii"), iterfield="in_file", name="fs2gii"
+        fs.MRIsConvert(out_datatype="gii",to_scanner=True), iterfield="in_file", name="fs2gii"
+    )
+    surfmorph_list = pe.Node(
+        niu.Merge(3,ravel_inputs=True),
+        name="surfmorph_list",
+        run_without_submitting=True,
+    )
+    fscurv2funcgii = pe.MapNode(
+        fs.MRIsConvert(out_datatype="gii",to_scanner=True,scalarcurv_file=True),iterfield="in_file", name="fscurv2funcgii"
+    )
+    allsurf_list = pe.Node(
+        niu.merge(7,ravel_inputs=True),
```

Comment:
is this a list of lists? if not, ravel_inputs isn't needed
```suggestion
        niu.Merge(7),
```